package.json

```diff
@@ -2,30 +2,22 @@
   "name": "libnpmpublish",
   "version": "2.0.0",
   "description": "Programmatic API for the bits behind npm publish and unpublish",
-  "author": {
-    "name": "Kat Marchán",
-    "email": "kzm@zkat.tech",
-    "twitter": "maybekatz"
-  },
+  "author": "Kat Marchán <kzm@sykosomatic.org>",
   "license": "ISC",
   "scripts": {
     "prerelease": "npm t",
     "release": "standard-version -s",
     "postrelease": "npm publish && git push --follow-tags",
     "pretest": "standard",
```

Comment:
Doesn't have to be done in this PR, but we've been moving these to this scripts setup:

```json
{
  "scripts": {
    "preversion": "npm test",
    "postversion": "npm publish",
    "prepublishOnly": "git push origin --follow-tags",
    "lint": "standard",
    "test": "tap",
    "posttest": "npm run lint"
  }
}
```